source/includes/api/_geostore.md

```diff
@@ -1,58 +1,71 @@
 
-Geostore is a geojson database. You can save your geojson or obtain geojson by country, region, Protected areas, etc.
+## What is Geostore
 
-Geostore object contains the following fields:
+Geostore is a database for storing geoStore objects which contain geographic data structures described using the [GeoJSON](https://geojson.org/) schema. GeoJSON supports the following geometry types: `Point`, `LineString`, `Polygon`, `MultiPoint`, `MultiLineString`, and `MultiPolygon`. Geometric objects with additional properties are `Feature` objects. Sets of features are contained by `FeatureCollection` objects.
 
+The Geostore microservice provides a comprehensive way to communicate geographic data structures between platforms and microservices. It enables users to [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) geographic data structures from the Geostore, and query information about a geoStore object, such as the bounding box of all its' geometries.
 
-|	 Field             | Description                                                       | Type
-| ------------------|:-----------------------------------------:                        | -----:
-| id                | Id of the geostore                                                | Text
-| geojson           | Geojson with the geometry                                         | Object
-| hash              | MD5 hash generated from geojson. Is the same that the id          | Object
-| provider          | This field should be completed if the geostore was created from a provider | Object
-| -- type           | Provider type                             | Text
-| -- table          | Table name                                | Text
-| -- user           | User of the account                       | Text
-| -- filter         | Conditions to obtain the geojson. It is possible use AND & OR conditions. This conditions must only return one row.   | Text
-| areaHa            | Area in Hectares of the geojson                 | Number
-| bbox              | Bounding box of the geojson                 | Array
+This documentation describes the Geostore microservice in two sections:
 
+- First we give an [overview of the available endpoints](#overview-of-available-endpoints) that allow you to [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) geoStore objects.
+- Next we describe the [structure and properties of Geostore objects](#geostore-objects-and-properties).
 
-<aside class="notice">
-Remember — All endpoint of geostore don't need that you are authenticated.
-</aside>
+After viewing the documentation below, consider looking at the Geostore tutorial for a step-by-step guide to rendering an actual geoStore object on a web application.
 
-## Create Geostore
+***Remember — All of the Geostore endpoints do not require that users are authenticated.***
```

Comment:
remove this please, let's not document things the user _doesn't_ need

Reply:
Removed.